stratum/hal/bin/barefoot/deb/start-stratum.sh

```diff
@@ -17,6 +17,8 @@ fi
 # Try to load the platform string if not already set.
```

Comment:
I think we said to stop sonic as part of this as well.

`sudo systemctl stop sonic.target`

Reply:
You are right, we should also disable the sonic.target(sonic services) before starting Stratum.
The thing is that we cannot run this in the container since the target is not here.
Or we can ignore the error for this command?

Reply:
Ignore the error in `start-stratum.sh` and add the same stop command to `start-stratum-container.sh`?